packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -265,17 +269,22 @@ const MetadataLabeling = (props: Props) => {
     }
 
     const onSubmit = async (value: string | undefined | null) => {
+        isSubscribedToSubmitResult.current = true;
+        setPending(true);
         const result = await addMetadata({
             ...props.payload,
             value: value || undefined,
         });
-        // todo: maybe some pending status?
-        if (result) {
-            setShowSuccess(true);
+        if (isSubscribedToSubmitResult.current) {
```

Comment:
compare pending string with current account prop

Reply:
in the [`892b94c` (#2596)](https://github.com/trezor/trezor-suite/pull/2596/commits/892b94c86110f19e1067830e4a2e55e4951fb354) I have changed  `isSubscribedToSubmitResult.current` to string